pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -212,6 +213,10 @@ func (vs *VSphere) HasClusterID() bool {
 func buildVSphereFromConfig(cfg *ccfg.CPIConfig, nsxtcfg *ncfg.Config, lbcfg *lcfg.LBConfig, routecfg *rcfg.Config) (*VSphere, error) {
 	nm := newNodeManager(cfg, nil)
 
+	if nsxtcfg == nil && (lbcfg != nil || routecfg != nil) {
```

Comment:
I think it would be better to indent the whole block here based on nsxtcfg being non-nil.
```
if nsxtcfg != nil {
    	ncm, err := nsxt.NewConnectorManager(nsxtcfg)
	if err != nil {
             ....
	}

        if lbcfg != nil {
              ....
        }

        if routcfg != nil {
               ....
        }
}
```

Reply:
+1